test/alignments/AlignmentsCollectionEntity.test.js

```diff
@@ -1,17 +1,19 @@
 import { AlignmentsCollectionEntity } from '../../src/alignments/AlignmentsCollectionEntity.js';
```

Comment:
This test wasn't running before, so you'll see a bunch of fixes below to get it working.